plugins/foreman_discovery/18.0/index.md

```diff
@@ -0,0 +1,946 @@
+
+The foreman-installer is able to automatically install the package and proceed
+with all necessary steps. To do this, re-run the installer with the following
+option:
+
+    # foreman-installer --enable-foreman-plugin-discovery
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case some changes have been made, proceed with manual
+download described below.
+
+### 2.1.2 Manual installation
+
+To install Foreman Discovery plugin, do the following
+
+    # yum install tfm-rubygem-foreman_discovery
+
+on Fedora and Red Hat systems or
+
+    # apt-get install ruby-foreman-discovery
+
+on Debian and Ubuntu systems.
+
+Restart of Foreman is needed when not using the installer:
+
+    # systemctl restart foreman
+
+## 2.2 Smart Proxy Discovery plugin (optional)
+
+Smart Proxy plugin can be used to discover hosts in networks Foreman can't
+directly reach itself. The plugin must be installed on all proxy hosts
+on the network where discovery feature is supposed to work.
+
+To install the plugin, do
+
+    # yum install rubygem-smart_proxy_discovery
+
+on Fedora and Red Hat systems or
+
+    # apt-get install ruby-smart-proxy-discovery
+
+on Debian and Ubuntu systems.
+
+Smart Proxy must be restarted after plugin installation and features must be
+refreshed for the given Proxy in the Foreman interface (*Infrastructure >
+Smart Proxies > a proxy > Refresh features*).
+
+## 2.3 Foreman Discovery Image
+
+The image is based on CentOS 7 and about 150 MB in size. There are two ways
+to install the image.
+
+### 2.3.1 Download via installer (recommended)
+
+The foreman-installer is able to automatically download latest stable image.
+For this, re-run the installer with the following option:
+
+    # foreman-installer \
+      --enable-foreman-proxy-plugin-discovery \
+      --foreman-proxy-plugin-discovery-install-images=true
+
+The command installs latest available Foreman Discovery Image. If that's not the version required (see the compatibility table above), explicitly request to download specific version:
+
+    # foreman-installer \
+      --enable-foreman-proxy-plugin-discovery \
+      --foreman-proxy-plugin-discovery-install-images=true \
+      --foreman-proxy-plugin-discovery-source-url=http://downloads.theforeman.org/discovery/releases/{{page.imgver}}/
+
+Tip: It is possible to install both Discovery plugin and image in one
+installer run by providing both the options.
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case some changes have been made, proceed with manual
+download described below.
+
+### 2.3.2 Manual download
+
+Images are available [for direct
+download](http://downloads.theforeman.org/discovery/)
+To download the latest release to the expected location, do the following on
+Fedora and Red Hat systems:
+
+    # wget http://downloads.theforeman.org/discovery/releases/{{page.imgver}}/fdi-image-latest.tar \
```

Comment:
```suggestion
    # wget https://downloads.theforeman.org/discovery/releases/{{page.imgver}}/fdi-image-latest.tar \
```

Reply:
Wont this fail on systems which dont have our CA?

Reply:
Yeah, it's a globally trusted CA.